src/main/js/components/TableView.jsx

```diff
@@ -51,23 +92,17 @@ class TableView extends React.Component {
     const savedView = either(localStorage)
       .map(f.prop(["tableViews"]))
```

Comment:
lodash supports mapping to an objects property but just naming the property. So `.map("tableViews")` should be enough.